### PR TITLE
Small typo and improvement on the k8s guide

### PR DESCRIPTION
**Pull request by loicmathieu**

### CODE REVIEW

File: docs/src/main/asciidoc/deploying-to-kubernetes.adoc [hunk 1]
```diff
@@ -667,13 +667,13 @@ After deploying these resources to Kubernetes, the Ingress resource will allow u
...
 
-[source]
+[source, properties]
 ----
 quarkus.kubernetes.ingress.expose=true
 # To change the Ingress host. By default, it's empty.
 quarkus.kubernetes.ingress.host=prod.svc.url
 # To change the Ingress path of the generated Ingress rule. By default, it's "/".
-quarkus.kubernetes.ports.http.path=/prod
+quarkus.kubernetes.ingress.path=/prod
```
Comment (Sgitario) [inline docs/src/main/asciidoc/deploying-to-kubernetes.adoc:676]: This property does not exist, the right one is `quarkus.kubernetes.ports.http.path`.

**Review by Sgitario — CHANGES_REQUESTED** [review]

File: docs/src/main/asciidoc/deploying-to-kubernetes.adoc (hunk 1 above)
Comment (loicmathieu) [reply docs/src/main/asciidoc/deploying-to-kubernetes.adoc:676]: Oh yes, I messed up with rule configuration.

By the way, it's not very easy to understand why a Kubernetes port have a path ?

Comment (Sgitario) [reply docs/src/main/asciidoc/deploying-to-kubernetes.adoc:676]: That's very true... We should address this not-very user-friendly property sooner than later.

Comment (loicmathieu) [reply docs/src/main/asciidoc/deploying-to-kubernetes.adoc:676]: OK, I can open an issue.
Should I close the PR or just remove this change ? Other change are cosmetic.

Comment (Sgitario) [reply docs/src/main/asciidoc/deploying-to-kubernetes.adoc:676]: For me, it's fine to just remove this change and proceed with the rest. 
And yes, feel free to open an issue to improve this configuration.

Comment (loicmathieu) [reply docs/src/main/asciidoc/deploying-to-kubernetes.adoc]: Line removed and issue create: https://github.com/quarkusio/quarkus/issues/29279

**Review by Sgitario — APPROVED** [review]

**Pull request merged**